chrome/content/zotero/zoteroPane.xul

```diff
@@ -299,6 +299,12 @@
 							<menuseparator/>
 							<menuitem class="menuitem-iconic zotero-menuitem-toggle-read-item" oncommand="ZoteroPane_Local.toggleSelectedItemsRead();"/>
 							<menuitem class="menuitem-iconic zotero-menuitem-duplicate-item" label="&zotero.items.menu.duplicateItem;" oncommand="ZoteroPane_Local.duplicateSelectedItem().done();"/>
+							<menu class="menuitem-iconic zotero-menuitem-add-to-collection">
+								<menupopup id="zotero-add-to-collection-popup" onpopupshowing="ZoteroPane_Local.buildAddToCollectionMenu(event)">
+									<menuseparator id="zotero-add-to-collection-separator"/>
+									<menuitem id="zotero-add-to-new-collection" label="&zotero.toolbar.newCollection.label;" oncommand="ZoteroPane_Local.addItemsToCollection(null, true)"/>
```

Comment:
Oh, nice — but let's move this above the hierarchical tree. I have a ton of collections in my test library and didn't even notice this existed until I looked at the code

Also can note in the description that this closes https://github.com/zotero/zotero/issues/2069